styles/variables-carbon.less

```diff
@@ -36,6 +36,7 @@
 @agate-heading-title-decoration-border-radius: 3px;
 @agate-heading-title-decoration-opacity: 0.8;
 @agate-heading-title-decoration-transform: skew(-10deg);
+@agate-heading-title-padding: 0 51px 21px 51px;
```

Comment:
can you use this value in variables-base? 
An remove the variables in the files specific for skins, since they all use the same value